homeassistant/components/climate/nest.py

```diff
@@ -97,6 +101,15 @@ def __init__(self, structure, device, temp_unit):
         self._min_temperature = None
         self._max_temperature = None
 
+    @property
+    def should_poll(self):
+        """Do not need poll thanks using Nest streaming API."""
+        return False
+
+    async def async_nest_update_callback(self):
+        """Update device state."""
+        await self.async_update_ha_state(True)
```

Comment:
We should never await this. Just call `self.self.async_schedule_update_ha_state(True)` (no need to await)

Reply:
@balloob It's the coroutine `async_update_ha_state` not the callback `async_schedule_update_ha_state`.